### ISSUE TITLE
PinSound not working in 10.8x64

### ISSUE THREAD

**Issue by kapter316** [vpinball/pinmame#197]
Hello, 
I am hoping you can assist. I saw another post where it looks like the issue was resolved, but I'm still experiencing it. I downloaded the latest: sc-3.6-720-e4df859-win-x86.zip and sc-3.6-720-e4df859-win-x64.zip. When running x64 test, PinSound works with no problems; you cannot get it to load with VPX10.8.8 32bits launcher. However, when I run x86 through a test or VPX10.8.8 64bits launcher, I always get the following error:  "unsupported pinball sound system." I have tried to run as admin and not as admin for all the programs above, and I still have the same issue. Is there a version of pinmame that runs PinSound in VPX10.8 64bits launcher? If so can you direct me to that version, or is this a bug? 
One last piece of information, the 10.7 32bits launcher will load and does play PinSound.  
I appreciate everything you do for us. 

Thank you,
Brian

**Comment by alaw-lufc** [comment]
Not sure if this is the same issue as noted here - [](https://www.vpforums.org/index.php?showtopic=48542&p=532879)
" I'm all 64-bit with my setup and all Bally/Williams pins work perfectly with altsound=2.  However, if I attempt the same process with Data East / Sega / Stern games then I get the "unsupported pinball sound system" error in PinSound Studio."

Some Pinsound mixes can work with x64, but some don't?

**Comment by kapter316** [comment]
> Not sure if this is the same issue as noted here - " I'm all 64-bit with my setup and all Bally/Williams pins work perfectly with altsound=2. However, if I attempt the same process with Data East / Sega / Stern games then I get the "unsupported pinball sound system" error in PinSound Studio."
> 
> Some Pinsound mixes can work with x64, but some don't?

Thank you for the comment. I retested, and you are correct; Pinsound now only works with Bally/Williams tables in 64-bit and works on almost all the others in 32-bit. I'm still hoping this can be corrected as all new tables and a lot of updated tables are being converted to 64bit, which means Pinsound cannot be used on those tables going forward that are not Bally/Williams.

**Comment by timdeporte** [comment]
Yes, this is not a problem caused by vpx but caused by vpinmame. I hope a fix will be available in the near future.

**Comment by griffindodd** [comment]
I'd like to confirm that I am experiencing this exact issue too.

Bally/Williams tables using altsound 2 work perfectly in VpinMame 3.6 x64
Stern, DE and Sega all hang at loading scripts on startup in x64, they dont have the issue in x32

Although not the same, this is very similar behavior to a past x64 bug when table startup would hang if PinSound Studio was not running.

Software: VPX 10.8GL x64, VPinMAME 3.6, B2S 2.1.3, FlexDMD 1.9.1, Freezy 2.3.1, Pinup Popper 1.5
Screens: PF: Sony X85K 4k 120hz Portrait, BG/FullDMD: 1080p 60hz Portrait, Topper: UHD 60hz Landscape 
Computer: Ryzen 9 5900X, 64GB, RTX 4080 Super, NVME, Win 11

**Comment by T3naciousC** [comment]
I can confirm same issue, Data East tables not working in x64
Do we think there's a chance this will ever get fixed? I very much prefer the sound quality of pinsound over g-sound...
